README.md

```diff
@@ -60,6 +60,31 @@ new Fingerprint2().get(function(result, components){
 });
 ```
 
+Or using Promise
+
+```js
+new Fingerprint2(options).getPromise()
+  .then(function([result, components]){
```

Comment:
Make the function signature `({result: ..., components: ...})`. This is equally simple to use in ES6 (`({ result, components }) => ...`) and it avoids magic numbers.